e2e_test/udf/python.slt

```diff
@@ -39,6 +46,26 @@ select gcd(25, 15, 3);
 ----
 1
 
+query I
+select series(5);
+----
+0
+1
+2
+3
+4
+
+# FIXME: support table function with multiple columns
+# query IT
+# select series2(5);
+# ----
+# 0 0
+# 1 1
+# 2 2
+# 3 3
+# 4 4
```

Comment:
It would be like 😇

```suggestion
# FIXME: support table function with multiple columns
# query IT
# select series2(5);
# ----
# (0,0)
# (1,1)
# (2,2)
# (3,3)
# (4,4)
```

Reply:
Got it 😇